adls-plugins/docs/AzureDataLakeStore-batchsink.md

```diff
@@ -19,8 +19,11 @@ Properties
 **fileSystemProperties:** A JSON string representing a map of properties
 needed for the distributed file system. (Macro-enabled)
 
-**schema:** Output schema for the JSON document.
+**schema:** Output schema for the avro format JSON document
 
+**outputFormat:** The format of output files
+
+**textDelimiter:** Field Delimiter if file output format is text
```

Comment:
Delimiter to place between fields. Only used by the text output format. Defaults to [whatever the default is]